css/css-scroll-snap/scroll-target-snap-002.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html>
+<meta charset="utf-8">
+<title>focus() and snap position with snapping on (y-axis)</title>
```

Comment:
The title says "focus()" but the test uses "scrollIntoView()"

Reply:
Fixed.